apple/macbook-pro/12-1/default.nix

```diff
@@ -7,6 +7,20 @@
     <nixpkgs/nixos/modules/hardware/network/broadcom-43xx.nix>
   ];
 
+  powerManagement = {
+    cpuFreqGovernor = lib.mkDefault "conservative";
```

Comment:
```suggestion
    # enable gradually increasing/decreasing CPU freq, rather than using "powersave", which would keep CPU freq at 0.8Ghz.
    cpuFreqGovernor = lib.mkDefault "conservative";
```